### DISCUSSION TITLE
glob definition

### DISCUSSION THREAD

**Discussion by mischw** [fox-it/acquire#64] [General]
I was playing around with acquire's glob feature and I noticed it behaves different from other implementations (Bash globs, Python's glob module). Is there a definition of how your implementation of the glob parameter exactly works?

**Comment by Schamper** [comment]
Apologies for the delayed reply, I've been both on holiday and away for a conference. 

Could you go into more detail on what exactly is different? As far as I'm aware, we should behave exactly the same as Python's `glob` module or `Path.glob` method.
